src/main/java/org/embl/mobie/lib/create/ImagesCreator.java

```diff
@@ -296,48 +296,47 @@ public void addOMEZarrImage( String uri,
                 imageDataFormat,
                 ThreadHelper.sharedQueue );
 
-        if ( ! isImageValid( imageData, projectCreator.getVoxelUnit() ) ) {
```

Comment:
It would be good to keep this check if possible? It ensures that the image units remains consistent throughout the entire project + rejects multi-channel images. Maybe keep it until you've added an extra menu to allow the user to select a specific channel?